MCDataProducts/inc/KalSeedMC.hh

```diff
@@ -95,6 +105,7 @@ namespace mu2e {
     float _wireTau; // threshold cluster distance to the wire along the perpedicular particle path
     float _strawDOCA; // signed doca to straw
     float _strawPhi; // cylindrical phi from -pi to pi with 0 in Z direction
+    TrkStrawHitProvenance _provenance; // origin/validity of MC info object
```

Comment:
If it's possible to do the default initialization here rather than in the c'tor initializer list, that would be better.  I don't know if that's possible.

Reply:
AFAIK the default values have to be designated in the constructor (maybe in the definition for argument-supplied fields, but definitely in the initializer list otherwise).

Reply:
What I was getting at is that for sufficiently simple types, you can initialize it in the same line as the declaration

_provenance = (static_cast<StringedDigiProvenance>(DigiProvenanceDetail::Simulation);

Then it is implicitly present in the initializer list of all c'tors.  What I am not sure about is whether this type is simple enough.  I bet it will be if you use the typedef I suggested in my recent review.